main/daily_update.py

```diff
@@ -67,7 +70,7 @@ def extract_contact_file(sf_api, sqlfile_name, platform_name):
     :param platform_name: salesforce or pardot
     :return: dataframe
     """
-    fd = open(os.path.abspath(os.path.join(os.pardir, 'sqlserver', sqlfile_name)), 'r')
+    fd = open(os.path.join(settings.path, 'sqlserver/{}'.format(sqlfile_name)), 'r')
     if platform_name == 'salesforce':
         sqlfile = fd.read().format(settings.OwnerId, settings.PardotUser)
     elif platform_name == 'pardot':
```

Comment:
what is happening here. Perhaps comment

Reply:
I added one path parameter in settings.py file which indicate the location where the current parent directory is.